doc/api/worker_threads.md

```diff
@@ -1038,6 +1038,33 @@ Calling `unref()` on a BroadcastChannel allows the thread to exit if this
 is the only active handle in the event system. If the BroadcastChannel is
 already `unref()`ed calling `unref()` again has no effect.
 
+## Class: `CPUProfileHandle`
```

Comment:
If we're planning to make this available on the `v8` module as well, I'd be inclined to just document it there tbh

Reply:
Done !